.env.sample

```diff
@@ -0,0 +1,2 @@
+DATABASE=YOUR_URL_HERE
```

Comment:
This file isn't supposed to be committed (I think you're also supposed to remove `.sample` from the end). Also, the values for `YOUR_URL_HERE` and `YOUR_PORT_HERE` should be the actual values lol

Reply:
This file is supposed to be commited, the real one isn't